src/cmdc/datasets/official/AK/__init__.py

```diff
@@ -0,0 +1,129 @@
+import pandas as pd
+import requests
+import re
+
+from cmdc.datasets.official.base import CountyData
+
+county_map = {
+    "Anchorage Municipality": 2020,
+    "Kenai Peninsula Borough": 2122,
+    "Kodiak Island Borough": 2150,
+    "Fairbanks North Star Borough": 2090,
+    "Southeast Fairbanks Census Area": 2240,
+    "Yukon-Koyukuk Census Area": 2290,
+    "Matanuska-Susitna Borough": 2170,
+    "Nome Census Area": 2180,
+    "North Slope Borough": 2185,
+    "Northwest Arctic Borough": 2188,
+    "Juneau City and Borough": 2110,
+    "Ketchikan Gateway Borough": 2130,
+    "Petersburg Borough": 2195,
+    "Prince of Wales-Hyder Census Area": 2198,
+    "Sitka City and Borough": 2220,
+    "Bethel Census Area": 2050,
+}
+
+DATE_RE = re.compile(r'.+last updated on (\d{2}/\d{2}/\d{4}).*')
+
+def _find_col_by_prefix(df, col_prefix):
+    cols = [x for x in list(df) if x.lower().startswith(col_prefix.lower())]
+    assert len(cols) == 1
+    return cols[0]
+
+
+class Alaska(CountyData):
+    table_name = "us_covid"
+    pk = '("vintage", "dt", "fips", "variable_id")'
+
+    def get_cases(self):
+        url = "https://www.arcgis.com/sharing/rest/content/items/867f802ce1624b46b40d2bd281490078/data"
+        cases_df = pd.read_excel(url, sheet_name="Table 2", skiprows=2)
+
+        bor_col = _find_col_by_prefix(cases_df, "borough")
+        comm_col = _find_col_by_prefix(cases_df, "community†")
+
+        # iterate row by row and collect the data we need...
+        need_to_find = list(county_map.keys())
+        out = []
+        finding = None
+        for ix in range(cases_df.shape[0]):
+            row = cases_df.iloc[ix, :]
+            if finding is None and row[bor_col] in need_to_find:
+                finding = row[bor_col]
+                continue
+
+            if finding is not None and row[comm_col].lower() == 'total':
+                # this is the row
+                out.append(dict(
+                    fips=county_map[finding],
+                    cases_total=row["All Cases"],
+                    hospital_beds_in_use_covid_total=row["Hospitalizations"],
+                    deaths_total=row["Deaths"],
+                    recovered_total=row["Recovered"],
+                    active_total=row["Active"]
```

Comment:
`active_total` and `recovered_total` are not in the `data.us_covid` table.

I'm confident that you've made this update on your end but it would be nice to provide the corresponding update in the schema file so that we both have an up-to-date reference (`schema/covid_data.sql`).